src/ctia/http/routes/crud.clj

```diff
@@ -164,13 +164,9 @@
  (s/fn [{{:keys [get-store]} :StoreService
          :as services} :- APIHandlerServices]
   (let [capitalized (capitalize-entity entity)
-        ;; Adding additional query params for ES Fulltext search
         search-q-params* (st/merge
                           search-q-params
-                          {(s/optional-key :query_mode)
-                           (describe FullTextQueryMode "Elasticsearch Fulltext Query Mode. Defaults to query_string")
-
-                           ;; We cannot name the parameter :fields, because we already have :fields (part
+                          {;; We cannot name the parameter :fields, because we already have :fields (part
```

Comment:
I leave this here for until I get the ES :fields parameter right (in a separate PR)